testing/testcafe/tests/dataGrid/columnChooser.ts

```diff
@@ -0,0 +1,45 @@
+/* eslint-disable @typescript-eslint/no-floating-promises */
+
+import { createScreenshotsComparer } from 'devextreme-screenshot-comparer';
+import { Selector } from 'testcafe';
+import url from '../../helpers/getPageUrl';
+import createWidget from '../../helpers/createWidget';
+import DataGrid, { CLASS } from '../../model/dataGrid';
+import { getData } from './helpers/generateDataSourceData';
+
+fixture.disablePageReloads`Column chooser`
+  .page(url(__dirname, '../container.html'));
+
+test('Column chooser screenshot', async (t) => {
+  const { takeScreenshot, compareResults } = createScreenshotsComparer(t);
+  const dataGrid = new DataGrid('#container');
+
+  await t
+    .click(dataGrid.getColumnChooserButton())
+    // act
+    .expect(await takeScreenshot('column-chooser', dataGrid.element))
+    .ok()
+    // assert
+    .expect(Selector(`.${CLASS.columnChooser}`).exists)
```

Comment:
Let's not use ```Selector``` directly in our tests.

It will be better if we add this to the DataGrid page object model
E.g something like this:
```typescript
// testing/testcafe/model/dataGrid/columnChooser.ts
import { Selector } from 'testcafe';

const CLASS = {
  columnChooser: 'dx-datagrid-column-chooser',
};

export default class ColumnChooser {
  element = Selector('body').find(`.${CLASS.columnChooser}`);
}
```

Add it to the DataGrid page object model:
```typescript
// testing/testcafe/model/dataGrid/index.ts

export default class DataGrid extends Widget {
  // other code
  getColumnChooser(): ColumnChooser {
    return new ColumnChooser();
  }
  // other code
}
```

And use it from the page object model:

```suggestion
    .expect(dataGrid.getColumnChooser().element).exists)
```